app/assets/v2/js/pages/bulk_payout.js

```diff
@@ -185,21 +185,20 @@ $(document).ready(function($) {
 });
 
 var get_total_cost = function() {
-  var num_rows = $('#payout_table tbody').find('tr').length;
   var total = 0;
-  var i = 1;
 
-  for (i = 1; i < num_rows; i += 1) {
-    var $row = $('#payout_table tobdy').find('tr:nth-child(' + i + ')');
-    var amount = parseFloat($row.find('.amount').text());
-    var username = $row.find('.username-search').text();
+  $('#payout_table tbody tr').each(function() {
+    var amount = parseFloat($(this).find('.amount').text());
+    var username = $(this).find('.username-search').text();
     var is_error = !$.isNumeric(amount) || amount <= 0 || username == '' || username == '@';
 
     if (!is_error) {
       total += amount;
     }
-  }
+  });
+
   return total;
```

Comment:
@Anish-Agnihotri okay. Sorry but `.each()` function won't be very helpful here because callback functions will be executed asynchronously, and hence when you return total outside, no guarantee that it will give the total.

Something like this will be good.
```js
var rows = $('#payout_table tbody tr');
for (i = 0; i < rows.length; i += `) {
    var $rows = $(rows[i]);
    var amount = parseFloat($row.find('.amount').text());
    // and all the other code
}
return total;
```

Reply:
Ah, yes, async execution. Thanks for the reviews, always a good learning experience. Pushing it out according to your recommendation now.

Reply:
@SaptakS @Anish-Agnihotri Hey guys, chiming in to clarify something.

> Sorry but .each() function won't be very helpful here because callback functions will be executed asynchronously,

jQuery `each` function is _synchronous_. You can verify this by adding a `console.log(total)` after the `each` function call. You can also refer to [this](https://stackoverflow.com/questions/7371942/is-jquery-each-function-synchronous) SO thread for more information.